### PR TITLE
Added Nord Theme

### PR DESCRIPTION
**Pull request by ChristosBouronikos**
I added a port of the famous Nord theme.

### CODE REVIEW

**Comment by abnerlee** [comment]
The thumbnail seems wired, could you retake a 500x400 or 250x200 thumbnail file?

**Comment by ChristosBouronikos** [comment]
Will update right away!

> The thumbnail seems wired, could you retake a 500x400 or 250x200 thumbnail file?

**Comment by ChristosBouronikos** [comment]
Better now?

**Comment by ChristosBouronikos** [comment]
There's still something wrong with the thumbnail at http://theme.typora.io/

**Edit:** Found what's wrong. On the website, the thumbnail links to "/media/thumbnails/nord.jpg"
It should be "/media/thumbnails/nord **.png** "

File: _posts/theme/2020-8-18-Nord.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+layout: theme
+title: Nord
+category: theme
+homepage: https://github.com/ChristosBouronikos/typora-nord-theme/
+download: https://github.com/ChristosBouronikos/typora-nord-theme/releases/download/1.0/typora-nord-theme.zip
+built-in: false
+author: Christos Bouronikos
+thumbnail: nord.jpg
```
Comment (abnerlee) [inline _posts/theme/2020-8-18-Nord.md:9]: you are using nord.png, but here you wrote jpg

Comment (ChristosBouronikos) [reply _posts/theme/2020-8-18-Nord.md:9]: dammit, thanks! will fix asap!

File: _posts/fork/2020-8-18-Nord.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+---
+layout: theme
+title: Nord
+category: theme
```
Comment (abnerlee) [inline _posts/fork/2020-8-18-Nord.md:4]: you don't need to submit two posts.

If your theme is forked from existing Typora theme, put it under `fork`
If not, put it under `theme`

Comment (abnerlee) [reply _posts/fork/2020-8-18-Nord.md:4]: You can also test in our local.

https://theme.typora.io/doc/Submit-A-Theme/

**Comment by abnerlee** [comment]
I revert this temporary. you may need to rebase with our origin, then submit a new PR

**Pull request merged**